<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPanel</class>
 <widget class="QWidget" name="ConfigPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="downloadPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下载文件位置选择：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadSet">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>并行任务数：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="maxTask">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="syncPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="synccSet">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>文件同步目录：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autosync">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>开启文件同步</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
